bin/dotd

```diff
@@ -236,6 +237,26 @@ def print_dtp_message
   end
 end
 
+# Retrieves recent Honeybadger issues, displaying them on the console.
+def get_honeybadger_issues
+  issues = Honeybadger.get_recent_issues
+
+  if issues.count > 0
+    puts ''
+  end
+  issues.each do |issue|
+    message = "(#{issue[:url]} [#{issue[:environment]}, #{issue[:project]}]) #{issue[:message]}"
```

Comment:
I often work in a half-screen terminal (about 110 characters wide) and this long line isn't going to parse too well.  I also tend to care less about the URL than I do about the environment/project/message.  I wonder if a two-line format would work better, even for messages that wrap or have newlines:
```
[production, dashboard] undefined method 'script_levels' for nil:NilClass (NoMethodError)
  ∟ https://app.honeybadger.io/projects/3240/faults/32696825
[staging, dashboard] An error has occurred, all later migrations canceled:

Mysql2::Error: Duplicate entry '2015-03-02' for key 'index_users_on_configuration_token...' etc. etc.
  ∟ https://app.honeybadger.io/projects/3240/faults/33396825
```